packages/components/src/__future__/Tag/RemovableTag/_docs/RemovableTag.stickersheet.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { RemovableTag } from "../index"
+
+export default {
+  title: "Components/Tag/Future/RemovableTag",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: () => (
+    <>
+      <StickerSheet heading="Button Group">
+        <StickerSheet.Header headings={["Default", "Hover", "Focus visible"]} />
+        <StickerSheet.Body>
+          <StickerSheet.Row>
+            <RemovableTag
+              color="gray"
+              removeButtonProps={{ ariaLabel: "Remove tag", onClick: () => {} }}
```

Comment:
These throw `eslint` errors for me, maybe just add an alert in there to silence them.